src/test/java/org/reaktivity/nukleus/maven/plugin/internal/generated/List8FW.java

```diff
@@ -0,0 +1,127 @@
+/**
+ * Copyright 2016-2019 The Reaktivity Project
+ *
+ * The Reaktivity Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package org.reaktivity.nukleus.maven.plugin.internal.generated;
+
+import org.agrona.BitUtil;
+import org.agrona.DirectBuffer;
+import org.agrona.MutableDirectBuffer;
+import org.agrona.concurrent.UnsafeBuffer;
+
+public final class List8FW extends ListFW
+{
+    final DirectBuffer fieldsRO = new UnsafeBuffer(0L, 0);
+
+    private static final int LENGTH_SIZE = BitUtil.SIZE_OF_BYTE;
+
+    private static final int FIELD_COUNT_SIZE = BitUtil.SIZE_OF_BYTE;
+
+    private static final int LENGTH_OFFSET = 0;
+
+    private static final int FIELD_COUNT_OFFSET = LENGTH_OFFSET + LENGTH_SIZE;
+
+    private static final int FIELDS_OFFSET = FIELD_COUNT_OFFSET + FIELD_COUNT_SIZE;
+
+    private static final int LENGTH_MAX_VALUE = Byte.MAX_VALUE + Byte.MAX_VALUE + 1;
```

Comment:
Let's put in the exact value for `LENGTH_MAX_VALUE` in hex, which is `0xff`, right?